assets/src/edit-story/components/inspector/prepublish/reviewChecklistDialog.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __ } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Text, THEME_CONSTANTS } from '../../../../design-system';
+import Dialog from '../../dialog';
+
+const TEXT = {
+  TITLE: __('Review checklist before publishing.', 'web-stories'),
+  SECONDARY_BUTTON: __('Continue to publish', 'web-stories'),
+  PRIMARY_BUTTON: __('Review Checklist', 'web-stories'),
+  BODY: __(
+    'Complete checklist items to make this Web Story easier to discover on Google Search.',
+    'web-stories'
+  ),
+};
+
+function ReviewChecklistDialog({ open, onIgnore, onReview, onClose }) {
```

Comment:
I think we use `isOpen` in other places. Maybe we could use that to be consistent? I guess the `Dialog` api uses `open` though so either works

I'm more used to `is`, `are` for boolean props, but it's not a big deal

Reply:
this is a hold over from the editor while we were updating to the new design system and we decided to create a dialog internal to the editor that sets up the most common patterns. totally agree with you but I think we should update 'em all at once.